### DISCUSSION TITLE
Can Daxie develop the web side?

### DISCUSSION THREAD

**Discussion by yi250** [abraunegg/onedrive#1594] [Q&A]
I download Chinese folders one-way. I don't know why they all download, but switch to two-way sync and you can download them normally. 

I still hope that Daxie will develop the web side. I really like your project, but the command line operation makes it very difficult for me to use. There's gonna be a lot of problems I can't handle.

**Comment by yi250** [comment]
Well, I found that this is a caching problem. As long as the previous process has not ended, the last task will be executed next time regardless of input. Is there any way to close it?

**Comment by abraunegg** [comment]
@yi250 
> I download Chinese folders one-way. I don't know why they all download, but switch to two-way sync and you can download them normally.

This depends on what client options you are using ... if you use `--download-only` or `--upload-only` you are doing a 1 way sync

> I still hope that Daxie will develop the web side.

What is 'Daxie' ?

> Well, I found that this is a caching problem. 

What caching problem? This client does not use 'caching' ... 

> As long as the previous process has not ended, the last task will be executed next time regardless of input. Is there any way to close it?

Well this depends on how you are starting the client, and what OS you are running on - as if Ubuntu - you have this issue too: https://github.com/abraunegg/onedrive/issues/1274
